extensions/ql-vscode/src/remote-queries/view/StarCount.tsx

```diff
@@ -12,6 +12,7 @@ const Count = styled.span`
   text-align: left;
   width: 2em;
   margin-left: 0.5em;
+  margin-right: 1.5em;
```

Comment:
My preference would still be to use css to do the layout and avoid empty DOM elements, but I will follow the layout that's already being done here and use `VerticalSpace`.

Reply:
Hmmm...`VerticalSpace` isn't working here because of how flexbox is being used. The `Star` element has a `flex-grow: 2;`, which means that it grabs relatively more space than other elements in the flex layout. Since `VerticalSpace` implicitly has `flex-grow: 0;`, it will shrink to its minimum width.

I _could_ avoid using `flex-grow` at all and use spacing dom elements for layout, but that sort of defeats the purpose of using flex-box.

Reply:
I've been using `Box` for layouts. Feel free to ignore my suggestion for now.